code/__DEFINES/dcs/signals.dm

```diff
@@ -40,6 +40,9 @@
 #define COMSIG_PARENT_QDELETING "parent_qdeleting"
 /// generic topic handler (usr, href_list)
 #define COMSIG_TOPIC "handle_topic"
+/// Trait signals
+#define COMPONENT_ADD_TRAIT (1<<0)
```

Comment:
why this define name? its not a component

Reply:
using numbers for signal values is not a great idea

Reply:
should likely just have SIGNAL_TRAIT_REMOVED(trait) and SIGNAL_TRAIT_ADDED(trait)

Reply:
`COMPONENT_(something)` is how we tend to define the signal-based flags.
Can do the change spooky suggests, though.

Reply:
but this isnt a flag its a signal?